src/main/java/com/github/manolo8/darkbot/backpage/BackpageManager.java

```diff
@@ -60,14 +62,18 @@ protected static class SidStatus {
     private int userId;
     private Optional<LoginData> loginData;
 
+    private Gson gson;
```

Comment:
Consider initializing it directly here instead of the constructor, makes it so the constructors on all the other apis automatically get it, and you can remove the null-check from getGson method
```suggestion
    private final Gson gson = new Gson();
```
I'd also move it to the top of the class.